fastlane/Pluginfile

```diff
@@ -2,4 +2,6 @@
 #
 # Ensure this file is checked in to source control!
 
+# Using ref: 8482a43 for files_with_version_number can still be an array
+# This is only temporary until main udpates files_with_version_number to be a hash 
```

Comment:
Is this still the case? Also
```suggestion
# This is only temporary until main updates files_with_version_number to be a hash 
```

Reply:
Oops, forgot to remove that comment 😅 Thanks!